android-app/app/src/main/java/com/nimtome/app/ui/theme/Theme.kt

```diff
@@ -29,9 +29,18 @@ private val LightColorPalette = lightColors(
 
 @Composable
 fun DndSpellsTheme(darkTheme: Boolean = isSystemInDarkTheme(), content: @Composable() () -> Unit) {
+    val systemUiController = rememberSystemUiController()
     val colors = if (darkTheme) {
+        // You can give your prefered color here, for the status bar(dark theme)
+        systemUiController.setSystemBarsColor(
```

Comment:
Could you please move these function calls out of this if expression?
Since this is some additional side effect that doesn't feel right here.
Instead please declare a separate function that handles the system bar color setting. That way the details of this system ui controller library is also abstracted away so its ETC later